src/calendar/actions/calendar/check-calendar-location.ts

```diff
@@ -0,0 +1,28 @@
+import { Fetcher } from '../../../common/api';
+import { endpoints } from '../../api/calendar/endpoints';
+import { ICheckLocationInputDto } from '../../types';
+import { errorFactory, CalendarErr } from '../../errors/errors';
+
+export const checkCalendarLocation = async (
```

Comment:
```suggestion
export const checkLocationForCalendar = async (
```

Est-ce que ce ne serait pas plus clair comme nom pour indiquer que l'on veut vérifier qu'une adresse est accessible selon les règles d'un calendrier donné ?